### PR TITLE
Added macOS Build instructions

### PR DESCRIPTION
**Pull request by paypay-a**
Added macOS Build instruction link back onto the readme

### CODE REVIEW

**Comment by paypay-a** [comment]
the workflow shouldnt affect anything since it doesn't exist anymore :)

**Comment by paypay-a** [comment]
i also have made some changes to the wiki on my repo if you wanna try and reflect those on this one

File: README.md [hunk 1]
```diff
@@ -60,8 +60,10 @@ If you want to contribute to the user interface translation project, please chec
 
 * __Windows__: [Windows Build](https://github.com/yuzu-mirror/yuzu/wiki/Building-For-Windows)
 * __Linux__: [Linux Build](https://github.com/yuzu-mirror/yuzu/wiki/Building-For-Linux)
+* __MacOS__: [MacOS Build](https://github.com/yuzu-mirror/yuzu/wiki/Building-For-MacOS)
+* __Android__: [Android Build](https://github.com/yuzu-mirror/yuzu/wiki/Building-For-Android)
 
-## Download
+* ## Download
```
Comment (ascpixi) [inline README.md:66]: Why was this header made a list entry?

**Review by ascpixi — CHANGES_REQUESTED** [review]
Thanks for the PR! I guess since the README was already modified, there's not much change to the parity of the repo, so we can get this through.

File: README.md (hunk 1 above)
Comment (paypay-a) [reply README.md:66]: Complete accident

**Comment by paypay-a** [comment]
Fixed the header being a list entry. Was a mistake i didnt catch before commiting it

**Comment by ascpixi** [comment]
Seems good, thank you!

**Pull request merged**